api/rosetta/networks.go

```diff
@@ -38,7 +44,7 @@ func (d *Data) Networks(ctx echo.Context) error {
 		return echo.NewHTTPError(http.StatusBadRequest, InvalidFormat(err.Error()))
 	}
 
-	// Get the network we are running on from the configuration.
+	// Get the current network from the configuration.
```

Comment:
Disagree with this change. You are changing the meaning here. `the network we are running on` is always going to be the same, while `current network` implies that the network could change (it can't).